packages/frontend-2/components/invite/dialog/Workspace.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <InviteDialogSharedSelectUsers
```

Comment:
There will be multiple modals here, it's just left out of this branch, which is an abstract of the branch which has all the modals